### ISSUE TITLE
The current MMR system is broken for old accounts

### ISSUE THREAD

**Issue by Arzybek** [ValveSoftware/Dota2-Gameplay#6511]
### Description

Hi! I'm playing Dota since 2013, I have one main account, I calibrated as soon as mmr system cam up to 2400, I climbed to 3200 max or something, than group and solo mmr got united, I got lowered to 2k or lower, was learning new position, lost more mmr, but it's ok I thought, the skill is always with me, right, but when I decided to climb back the problems begin... The system thinks since my account is old and I play good every game, it needs to match against me with smurf/booster every game, you can say whatever you want, but the players on 400 mmr shouldn't be playing 22/0 and 22/1 in almost every game, it's almost impossible even for me to climb back up, I'm not even talking about new players, and it's been like this for over 4-5 years now or something like that, I'm complaining on Reddit the whole time but no-one takes that into attention, can you fix the matchup for old/good players, it's not possible that games on 400 mmr should be harder than ones on 3k, I'm very upset just because I don't want to switch accounts and play on the new one, but it seems like the devs completely don't care about this issue and the only solution is to create new account, maybe  that's why almost no pro players is playing on their original account, didn't ever think about that? I can provide all the info and proofs if you contact me privately, thanks.

**Comment by B54Carey2** [comment]
This wall of text doesn't mean too much without a dotabuff to examine. Sure you might be right but random reddit users can't determine if you're the problem or if the quality of your matches is problematic without some evidence to look at.

**Comment by Arzybek** [comment]
> This wall of text doesn't mean too much without a dotabuff to examine. Sure you might be right but random reddit users can't determine if you're the problem or if the quality of your matches is problematic without some evidence to look at.

Yes, as I said, I will send dotabuff and all the examples for devs, don't want to publicly put them here

**Comment by WilliamElio** [comment]
The current MMR system is broken for old accounts because the current system was designed with the intent of providing an accurate MMR score for players that are relatively new to the game. This system is not effective for players who have been playing the game for a longer period of time, as the current MMR system does not take into account a players performance in the past and instead focuses on their recent performance. This means that players with an old account can easily have a low MMR score despite having a long history of good performance in the game. To fix this issue, the current MMR system should be overhauled to take into account a player's performance over a longer period of time[.](https://kipasguys.net/)

**Comment by AslanSmith** [comment]
Hey there, fellow Dota veteran! I totally feel your frustration with the match struggle. It’s hard when you’re giving your best, but the system throws some real curveballs your way. Your journey speaks to a sentiment shared by many of us – fighting the Smurf Boosters is quite a challenge I agree that inconsistent skills can be more than frustrating.

I've seen Reddit flooded with similar comments, and it's a bummer that the devs might not be as responsive as we expected. But hang in there, the Dota community is still growing, and your passion for the game shines through. Keep pushing, learning, and remember that your skills are indeed always with you. It will show your dedication in your first or new story. Keep the Dota spirit alive and well!   [.](https://gbinsta.dev/)
